activerecord/lib/active_record/connection_adapters/mysql/quoting.rb

```diff
@@ -2,8 +2,6 @@ module ActiveRecord
   module ConnectionAdapters
     module MySQL
       module Quoting # :nodoc:
-        QUOTED_TRUE, QUOTED_FALSE = '1', '0'
-
```

Comment:
Should we return freezing value for these quoted methods as well?
https://github.com/rails/rails/blob/v5.0.0.rc1/activerecord/lib/active_record/connection_adapters/abstract/quoting.rb#L114-L128

```ruby
      def quoted_true
        "'t'"
      end

      def unquoted_true
        't'
      end

      def quoted_false
        "'f'"
      end

      def unquoted_false
        'f'
      end
```